docs/csharp/misc/cs0043.md

```diff
@@ -1,13 +1,17 @@
 ---
 title: "Compiler Error CS0043"
 ms.date: 07/20/2015
-f1_keywords: 
+f1_keywords:
   - "CS0043"
-helpviewer_keywords: 
+helpviewer_keywords:
   - "CS0043"
 ms.assetid: 471e2cf8-4275-4618-ad20-408f829d2119
 ---
 # Compiler Error CS0043
-PDB file 'file' has an incorrect or out-of-date format. Delete it and rebuild.  
-  
+
+PDB file 'file' has an incorrect or out-of-date format. Delete it and rebuild.
+
  Delete the .pdb file for this compilation and recompile.
+
+> [!NOTE]
+> This compiler error is no longer used in Roslyn.
```

Comment:
Nice to have (don't know if it's worth the effort): indicate the C# version, starting with which the error is not generated anymore.

Reply:
I think it's not used in Roslyn from the beginning. It was used by the old compiler before Roslyn.

Reply:
https://github.com/dotnet/roslyn/releases

@pkulikov , Seems like Roslyn versioning is independent of C# language versioning, but it's being pushed in parallel with visual studio.

Reply:
@Youssef1313 thanks, indeed, it looks that's more about Roslyn versions rather than the C# versions.